src/Defaults.js

```diff
@@ -2,5 +2,6 @@ export default {
 	baseUrl: 'https://api.particle.io',
 	clientSecret: 'particle-api',
 	clientId: 'particle-api',
-	tokenDuration: 7776000 // 90 days
+	tokenDuration: 7776000, // 90 days
+	auth: undefined
```

Comment:
fwiw, i always wished this was named `token` just for clarity but i get that the ship has sailed there given the method arg object field names